yapapi/executor/__init__.py

```diff
@@ -581,3 +602,5 @@ async def __aexit__(self, exc_type, exc_val, exc_tb):
             emit(events.ShutdownFinished())
         except Exception:
             emit(events.ShutdownFinished(exc_info=sys.exc_info()))
+        finally:
+            await self._wrapped_consumer.stop()
```

Comment:
This hopefully prevents the occasional "Task was destroyed but is still pending" error